playbooks/install_local_api_dependencies.yml

```diff
@@ -1,8 +1,25 @@
 ---
-- name: local-api dependecies
+- name: local-api dependecies and build
   hosts: raspberry_masters
 
   roles:
     - role: geerlingguy.docker_arm
       become: true
-      tags: docker
+      tags: docker_installation
+
+  tasks:
+    - name: create .env file for main rpi (needed for monitor_pi.py)
+      ansible.builtin.copy:
+        dest: /home/pi/pyro-engine/pyroengine/pi_utils/.env
+        content: |
+          MAIN_RPI_IP=127.0.0.1
+          MAIN_RPI_WEBSERVER_PORT=8002
+
+    - name: run docker-composer.yml (takes around 10 to 15 minutes)
```

Comment:
Even with the last image from mateo hub ? 
About how long ?

Reply:
Did not try it, as I say, I did not touch this PR since it was ready for review because I think that the integration of the changes in pyroengine since 5/6 days ago will change stuff.